### PR TITLE
[RFC] doc; minor runtime bug fixes

### PR DESCRIPTION
**Pull request by justinmk**
todo

- [x] https://github.com/neovim/neovim/issues/7058#issuecomment-317203694

### CODE REVIEW

File: src/nvim/api/vim.c [hunk 1]
```diff
@@ -127,6 +127,9 @@ void nvim_feedkeys(String keys, String mode, Boolean escape_csi)
 /// Unlike `nvim_feedkeys`, this uses a lower-level input buffer and the call
 /// is not deferred. This is the most reliable way to emulate real user input.
```
Comment (bfredl) [inline src/nvim/api/vim.c]: _implement_ user input, not "emulate". Or just "send user input". The API _does_ things, it doesn't emulate doing things.

Comment (justinmk) [reply src/nvim/api/vim.c]: I'll change it, though I sympathize with the original wording: I think it was to contrast e.g. feedkeys() which is not equivalent to raw user input.

Comment (bfredl) [reply src/nvim/api/vim.c]: I didn't argue against that part. Removing "emulate" makes that point stronger, it indicates something slightly removed from the real thing, when it is in fact the real thing.

File: runtime/doc/options.txt [hunk 2]
```diff
@@ -6760,7 +6760,7 @@ A jump table for the options with a short description can be found at |Q_op|.
 	determined by the last focused window in the tabpage.  Highlights of
 	the popupmenu are determined by the current window.  Highlights in the
 	message area are not overridable.  Example for overriding the
-	backgrond color: >
```
Comment (bfredl) [inline runtime/doc/options.txt]: Also `builting` on line 6754 :P

Comment (justinmk) [reply runtime/doc/options.txt]: fixed in https://github.com/neovim/neovim/pull/7364

**Pull request merged**